apps/condo/domains/user/schema/SigninOrRegisterResidentUserService.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * Generated by `createservice user.SigninOrRegisterResidentUserService`
+ */
+const { getSchemaCtx } = require('@core/keystone/schema')
+const { GQLCustomSchema } = require('@core/keystone/schema')
+const access = require('@condo/domains/user/access/SigninOrRegisterResidentUserService')
+const {
+    CONFIRM_PHONE_ACTION_EXPIRED,
+} = require('@condo/domains/user/constants/errors')
+const { ConfirmPhoneAction, User } = require('@condo/domains/user/utils/serverSchema')
+
+const SigninOrRegisterResidentUserService = new GQLCustomSchema('SigninOrRegisterResidentUserService', {
+    types: [
+        {
+            access: true,
+            type: 'input SigninOrRegisterResidentUserInput { dv: Int!, sender: JSON!, token: String! }',
+        },
+        {
+            access: true,
+            type: 'type SigninOrRegisterResidentUserOutput { item: User, token: String! }',
+        },
+    ],
+
+    mutations: [
+        {
+            access: access.canSigninOrRegisterResidentUser,
+            schema: 'signinOrRegisterResidentUser(data: SigninOrRegisterResidentUserInput!): SigninOrRegisterResidentUserOutput',
```

Comment:
why should we use 1 mutation for signin and register? i'm not sure this is good idea

Reply:
This is to satisfy mobile app developers. They only have 1 screen to login and register

Reply:
Not sure if the UI should affect the API, it's better to separate the handles.